vtex/faststore
# Feat: Adds `RegionBar` and `RegionModal` Sections

## What's the purpose of this pull request?

Create CMS sections for `RegionBar` and `RegionModal`. Also integrate the CMS data.

## How it works?

- [x] Creates sections in `sections.json` for `RegionBar` and `RegionModal`.
- [x] adapt `GlobalSection` to receive CMS props.

After the merge
- [ ] sync in main workspace after merge this PR.
- [ ] publish changes in Global Sections content type in main workspace.

## How to test it?

- Run the core package locally: `yarn dev`
- Use my dev workspace in draft version  of `Global Section` page, and add a new `RegionBar` and `RegionModal` Sections: https://formspace--storeframework.myvtex.com/admin/new-cms/faststore/globalSections/edit/a56db112-87a6-40d5-a4d5-f6df686f93d0/
- Check the preview.

### Starters Deploy Preview

TBD after the merge of this PR
